packages/toolbox-core/src/toolbox_core/auth_methods.py

```diff
@@ -89,7 +92,15 @@ def _update_cache(new_token: str) -> None:
         raise ValueError(f"Failed to validate and cache the new token: {e}") from e
 
 
-def get_google_token_from_aud(audience: Optional[str] = None) -> str:
+def get_google_token_from_aud(
+    clock_skew_in_seconds: int, audience: Optional[str] = None
```

Comment:
this should be optional right? 
```suggestion
    clock_skew_in_seconds: int = 0, audience: Optional[str] = None
```

Reply:
We're already passing a default value from the `get_google_id_token` method, so, this is not needed. But adding for ease anyway.